spec/controllers/items_controller_spec.rb

```diff
@@ -71,8 +71,10 @@
   end
   describe 'embargo_update' do
     it 'should 403 if you are not an admin' do
-      allow(@current_user).to receive(:is_admin?).and_return(false)
-      post 'embargo_update', :id => @pid, :date => '12/19/2013'
+      expect(controller).not_to receive(:save_and_reindex)
+      expect(controller).not_to receive(:flush_index)
+      expect(controller).to receive(:forbid_modify).and_return(false)
```

Comment:
That's probably irrelevant for this spec.  (What might be more important is that this PR fixes the param name from :date to :embargo_date.)  But, if it must be perfect, name the date and you shall have it ;-)